components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md

```diff
@@ -246,7 +246,10 @@ The bastion server can patch new configurations in all nodes. To know more see [
 - For the Frontend nodes you can use the same IP in Chef Automate and Chef Server.
 - For the Backend nodes you can use the same IP in PostgreSQL and OpenSearch.
 - To provide multiline certificates use triple quotes like `""" multiline certificate contents"""`.
-
+- Rebooting or restarting individual nodes outside of a designated maintenance window especially during periods of high traffic should be avoided.
```

Comment:
Replace it with: - Rebooting or restarting individual nodes outside a designated maintenance window should be avoided, especially during periods of high traffic.